src/stitching/addSimpleRoutingResolvers.ts

```diff
@@ -0,0 +1,66 @@
+import { mapValues, isEmpty } from 'lodash';
+import { printSchema, print, ExecutionResult } from 'graphql';
+import { GraphQLFieldResolver, GraphQLSchema } from 'graphql';
+import { makeExecutableSchema } from '../schemaGenerator';
+
+type ResolverMap = { [key: string]: GraphQLFieldResolver<any, any> };
+
+export type Fetcher = (
+  operation: {
+    query: string;
+    operationName?: string;
+    variables?: { [key: string]: any };
+  },
+) => Promise<ExecutionResult>;
+
+export default function addSimpleRoutingResolvers(
+  schema: GraphQLSchema,
+  fetcher: Fetcher,
+): GraphQLSchema {
+  const queries = schema.getQueryType().getFields();
+  const queryResolvers: ResolverMap = mapValues(queries, (field, key) =>
+    createResolver(fetcher, key),
+  );
+  let mutationResolvers: ResolverMap = {};
+  const mutationType = schema.getMutationType();
+  if (mutationType) {
+    const mutations = mutationType.getFields();
+    mutationResolvers = mapValues(mutations, (field, key) =>
+      createResolver(fetcher, key),
+    );
+  }
+
+  const resolvers: {
+    Query: ResolverMap;
```

Comment:
Remote schemas are not working if they have root types that are not called `Query` or `Mutation`, such as
```schema {
  query: RootQuery
  mutation: RootMutation
}```

b/c the schema actually needs to be transformed as well to replace `RootQuery` => `Query` etc